src/protagonist/DLCS.Repository/Assets/AssetQueryX.cs

```diff
@@ -63,4 +64,48 @@ private static LambdaExpression CreateExpression(Type type, string propertyName)
 
         return Expression.Lambda(body, param);
     }
+
+    public static IQueryable<Asset> ApplyAssetFilter(this IQueryable<Asset> queryable, 
+        AssetFilter? assetFilter, bool filterOnSpace = false)
```

Comment:
`bool filterOnSpace = false` is always `false`, is it worth removing?

Reply:
It won't be in the two other calls that also take a query